po/ca.po

```diff
@@ -1530,80 +1530,6 @@ msgstr "Voleu restablir els colors personalitzats d'aquesta pestanya?"
 msgid "New tabs appear after the current tab"
 msgstr "Les noves pestanyes apareixen després de la pestanya actual"
 
-#: guake/prefs.py:186
```

Comment:
Why is all this deleted? Have you not updated your repo to master yet?

Reply:
I did rebase from master. I takes a lot of effort as well, to get Guake to a stable version. I feel the same that you guys keep maintaining a broken master, and that my branch has the good things that make a good guake. I am faced with the question of having guake prs_combined self-compiled or get the default /apt/dnf old 3.2 version with lack of real useful functions.

Reply:
It may be significantly easier to create branches from Guake master, and cherry pick your changes into the branches. Try to keep it one feature per branch, and open a seperate PR for each feature so that we can check them all feasibly. It'll be easier to avoid merge conflicts too since you'll be starting from master.